euth/dashboard/views.py

```diff
@@ -75,10 +75,25 @@ def get_success_url(self):
 
 
 class DashboardProjectUpdateView(DashboardBaseMixins,
+                                 SuccessMessageMixin,
                                  generic.UpdateView):
     model = project_models.Project
     form_class = forms.ProjectForm
     template_name = 'euth_dashboard/project_form.html'
+    success_message = _('Project successfully updated.')
+
+    @functional.cached_property
```

Comment:
This is an UpdateView for project. It already has the project in scope. It is called `self.object` or `self.get_object` (not sure).